css-layout-classes/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>CSS Layout Classes</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <h1>Choose your character!</h1>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-third">
+          <img src="images/link.png" alt="Link">
+        </div>
+        <div class="column-one-third">
+          <h2>Super Smash Bros. Ultimate</h2>
+          <p>Super Smash Bros. Ultimate received universal
+            acclaim from both critis and players, with some
+            critics calling ti the best installment in the
+            series. It received priase for its large amount
+            of content and fine-tuning of existing smash
+            gameplay elements, although its online mode was widely
+            criticized.
+          </p>
+        </div>
+        <div class="column-one-third">
+          <img src="images/yoshi.png">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-fourth">
+          <img src="images/kirby.png">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/fox.png">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/pikachu.png">
+        </div>
+        <div class="column-one-fourth">
+          <img src="images/ness.png">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-half">
```

Comment:
nice work, the only update you want is that the images for Donkey Kong and Mario in the last two rows should be 1/3 and the text should be 2/3. this will more closely align with the example image.

Reply:
https://user-images.githubusercontent.com/60913524/168175228-326c5bbb-4106-4cd2-821e-1a4235d7fcb4.mov